pipeline/outputs/s3.md

```diff
@@ -282,6 +289,23 @@ Example:
 
 Then, the records will be stored into the MinIO server.
 
+## Usage for Parquet Compression
+
+For parquet compression, it needs to install [columnify](https://github.com/reproio/columnify) in the running system or container at runtime.
```

Comment:
I'd say, "users need to install"

Can we flesh this out into an example with the FLuent Bit image so folks can easily figure it out?

From reaing this, I don't know how to use parquet quickly